src/video/wayland/SDL_waylandevents.c

```diff
@@ -633,9 +707,15 @@ seat_handle_capabilities(void *data, struct wl_seat *seat,
     }
 }
 
+static void
+seat_handle_name(void *data, struct wl_seat *seat, char *name)
+{
+    printf("seat has name %s\n", name);
```

Comment:
We might want to move this to go through the `SDL_Log` infra

Reply:
This was mostly just for proto and can probably be stripped out